core/templates/domain/exploration/WrittenTranslationsObjectFactory.ts

```diff
@@ -66,9 +67,9 @@ export class WrittenTranslations {
   }
 
   markAllTranslationsAsNeedingUpdate(contentId: string): void {
-    var languageCodeToWrittenTranslation = (
+    let languageCodeToWrittenTranslation = (
```

Comment:
Here and below: If we aren't reassigning values, prefer `const`.

Reply:
Made these changes in all the files, thanks for reviewing!